shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.test.js

```diff
@@ -132,6 +139,17 @@ describe('generatePrintablePendingReportInteractor', () => {
         documentTitle: 'Test Document Title',
         receivedAt: '2020-03-03T12:00:00.000Z',
       },
+      {
```

Comment:
Since this item is a bugfix, I'd love to see a new named test (ex - 'should sort pending items by date received, ascending) for that bug case.